Microservices/Repositories/IUnitOfWorkRepository.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Microservices.Repositories
+{
+    public interface IUnitOfWorkRepository<TEntity> : IRepository<TEntity>, IRepositoryAsync<TEntity>
```

Comment:
What is UnitOfWorkRepository?

Reply:
Here I used pattern unit of work, for different sync & async operations.

Reply:
Let's discuss it offline.